views/applications/one.html

```diff
@@ -0,0 +1,26 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{config.actions.edit|capitalize}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+  </div>
+  {% include "applications/_form.html" %}
+{% endblock %}
```

Comment:
I'm OK leaving pagination out of this PR until we figure out a more universal solution for for paginating the tables we have, but the Client ID and Client Secret are hard requirements for view; they're literally the thing we need to get from the webhooks from a user perspective.